src/modules/vendor/components/companies/ProfileInfo.vue

```diff
@@ -5,17 +5,97 @@
         <ni-input caption="Nom" v-model.trim="company.name" @focus="saveTmp('name')" @blur="updateCompany('name')" />
       </div>
     </div>
+    <div class="q-mb-xl">
+      <p class="text-weight-bold">Utilisateurs</p>
+      <q-card>
+        <ni-responsive-table :data="users" :columns="usersColumns" :pagination.sync="usersPagination">
+          <template v-slot:body="{ props }">
+            <q-tr :props="props">
+              <q-td v-for="col in props.cols" :key="col.name" :data-label="col.label" :props="props" :class="col.name"
+                :style="col.style">
+                <template v-if="col.name === 'actions'">
+                  <div class="row no-wrap table-actions">
+                    <div class="row no-wrap table-actions">
+                      <q-icon color="grey" name="edit" @click.native="openUserEditionModal(props.row)" />
+                      <q-icon color="grey" name="delete" @click.native="validateUserDeletion(col.value)" />
+                    </div>
+                  </div>
+                </template>
+                <template v-else>{{ col.value }}</template>
+              </q-td>
+            </q-tr>
+          </template>
+        </ni-responsive-table>
+        <q-card-actions align="right">
+          <q-btn no-caps flat color="primary" icon="add" label="Ajouter un utilisateur"
+            @click="userCreationModal = true" />
+        </q-card-actions>
+      </q-card>
+    </div>
+
+    <ni-modal v-model="userCreationModal" @hide="resetUserCreationForm" @show="openUserCreationModal">
+      <template slot="title">
+        Ajouter un <span class="text-weight-bold">utilisateur</span>
+      </template>
+      <ni-input in-modal v-model="newUser.identity.firstname" caption="Prénom" />
+      <ni-input in-modal v-model="newUser.identity.lastname" :error-label="REQUIRED_LABEL"
+        :error="$v.newUser.identity.lastname.$error" caption="Nom" @blur="$v.newUser.identity.lastname.$touch"
+        required-field />
+      <ni-input in-modal v-model="newUser.local.email" :error="$v.newUser.local.email.$error" caption="Email"
+        @blur="$v.newUser.local.email.$touch" :error-label="emailError($v.newUser)" required-field />
+      <ni-input in-modal v-model.trim="newUser.contact.phone" :error="$v.newUser.contact.phone.$error"
+        caption="Téléphone" @blur="$v.newUser.contact.phone.$touch" :error-label="phoneNbrError($v.newUser)" />
+      <ni-select in-modal caption="Role" :options="roleOptions" v-model="newUser.role" :error-label="REQUIRED_LABEL"
+        :error="$v.newUser.role.$error" @blur="$v.newUser.role.$touch" last required-field />
+      <template slot="footer">
+        <q-btn no-caps class="full-width modal-btn" label="Ajouter un utilisateur" icon-right="add" color="primary"
+          :loading="loading" @click="createUser" :disable="$v.newUser.$anyError || !$v.newUser.$anyDirty"/>
```

Comment:
quelle est la difference entre `$v.newUser.$anyError` et `$v.newUser.$invalid`

Reply:
dans notre cas `$invalid` est  le mieux, bien vu ! 
https://css-tricks.com/form-validation-in-under-an-hour-with-vuelidate/#article-header-id-0